components/dashboard/cardCharacterContainer/cardCharacterExibs/CardsCharacterExibs.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-    <div class="flex w-full flex-wrap md:justify-start justify-center gap-2">
-      <div v-for="character in charactersList" v-bind:key="character.id">
-        <CardCharacter :character="character" />
-      </div>
+  <div class="flex w-full flex-wrap md:justify-start justify-center gap-2">
+    <div v-for="character in charactersList" v-bind:key="character.id">
+      <CardCharacter :character="character" />
     </div>
+  </div>
 </template>
 
 <script>
-  import CardCharacter from '@/components/dashboard/cardCharacterContainer/cardCharacterExibs/cardCharacter/CardCharacter.vue'
+import CardCharacter from "@/components/dashboard/cardCharacterContainer/cardCharacterExibs/cardCharacter/CardCharacter.vue";
 
-  export default {
-    components: { CardCharacter },
-    name: "CardsCharacterExibs",
-    props: ["charactersList"],
-    }
-</script>
\ No newline at end of file
+export default {
+  components: { CardCharacter },
+  name: "CardsCharacterExibs",
+  props: ["charactersList"],
```

Comment:
Apenas "characters" já é o suficiente. Nomes no plural tendem a ser arrays/coleções.
```suggestion
  props: ["characters"],
```